t/058-tcp-socket.t

```diff
@@ -3298,6 +3299,7 @@ failed to receive a line: closed []
 close: 1 nil
 --- no_error_log
 [error]
+--- skip_eval: 3: system("ping6 -c 1 ::1 >/dev/null 2>&1") ne 0
```

Comment:
can we merge my dirty hack for now?
we can polish it later if we will have enough time and will